### DISCUSSION TITLE
Upgrade issue

### DISCUSSION THREAD

**Discussion by ahmahdy** [opendcim/openDCIM#1463] [Q&A]
I am trying to upgrade opdedcim from 23.01 to 23.04.
I copy the folder, change the symbolic links, and copy the db file and update it but when I try to navigate to the website. the website is stuck in a loop https://servername/install.php. FYI I upgraded this server before but I am not sure what am I doing wrong here.

**Comment by Tarot67** [comment]
Yes i have the same issue. Just stop opendcim and re-restore your dump (23.01) in your new mariadb drop table fac_Country. After that start opendcim 23.04 normally it is ok!. But that take a little bit time before opendcim up juste key "F5" severale time that take some time that depending of your config but after it is ok!.

**Comment by ediestajr** [comment]
Please check /var/log/php-fpm/www-error.log. @Tarot67 is correct, hit f5/refresh and monitor www-error.log
